### ISSUE TITLE
Add to F-Droid?

### ISSUE THREAD

**Issue by Perflyst** [mohamedebrahim96/PlexMedia#3]
Is there any chance that you can add this app to F-Droid?
F-Droid is a store for open source apps: https://f-droid.org/

**Comment by mohamedebrahim96** [comment]
yes, i would like to add it there ...... but I don't how to upload on F droid

**Comment by Perflyst** [comment]
Here is a how to, pretty straight forward: https://gitlab.com/fdroid/fdroiddata/blob/master/CONTRIBUTING.md
